samples/web/blazor-server-application/Core_8/WebApp/Program.cs

```diff
@@ -0,0 +1,38 @@
+using NServiceBus;
+
+#region ApplicationStart
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Host.UseNServiceBus(context =>
+{
+    var endpointConfiguration = new EndpointConfiguration("Samples.Blazor.WebApplication");
+    endpointConfiguration.MakeInstanceUniquelyAddressable("1");
+    endpointConfiguration.EnableCallbacks();
+    endpointConfiguration.UseTransport(new LearningTransport());
+    return endpointConfiguration;
+});
+
+#endregion
+
+builder.Services.AddRazorPages();
+builder.Services.AddServerSideBlazor();
+
+var app = builder.Build();
+
+if (!app.Environment.IsDevelopment())
+{
+    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+    app.UseHsts();
+}
```

Comment:
It's a sample, no need to handle any kind of production-like scenario

```suggestion
```

Reply:
Oh ok.  This was part of the default project files when creating a Blazor project.  I wasn't sure what needed to be cleaned up from that.